lesson13.md

```diff
@@ -0,0 +1,169 @@
+<!-- .slide: id="lesson13" -->
+
+# Basic Frontend - Fall 2021
+
+Lesson 13, Thursday, 2021-11-04
+
+---
+
+### Recap
+
+---
+
+### getElementById
+
+HTML:
+```html
+<input id="myInput">
+```
+
+JS:
+```js
+let myInputElement = document.getElementById("myInput");
+```
+
+What JavaScript code do I need to write to change the input element to:
+
+```html
+<input id="myInput" type="number">
+
+---
+
+HTML:
+```html
+<input id="myInput" type="number">
+```
+
+JS:
+```js
+let myInputElement = document.getElementById("myInput");
+myInputElement.type = "number";
+```
+
+---
+
+HTML:
+```html
+<input id="myInput" type="number" min=0 max=99 placeholder="Amount">
+```
+
+JS:
+```js
+let myInputElement = document.getElementById("myInput");
+myInputElement.type = "number";
+myInputElement.min = 0;
+myInputElement.max = 99;
+myInputElement.placeholder = "Amount";
+```
+
+---
+
+HTML
+
+```html
+<button id="myButton" onclick="myFunction()">Click me!</button>
+```
+
+How can we set the `onclick` property from JavaScript?
+
+```js
+let myButtonElement = document.getElementById("myButton");
+// ???
+```
+
+---
+
+### Variables pointing to functions
+
+So far, we learned that variables can point to values:
+
+```js
+let age = 42;
+let name = "Otto";
+let isHappy = true;
+let address = { street: "Ottostr.", city: "Berlin" }
+```
+
+---
+
+### Functions can be values, too
+
+Variables can also point to functions:
+
+```js
+let myFunction = function() { console.log("this function was called"); };
```

Comment:
I guess we have to explain anonymous functions here, maybe worth having it written on this slide.